stock_prices/stock_prices.py

```diff
@@ -3,13 +3,27 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
 
+  low = prices[0]
+  high = prices[0]
 
-if __name__ == '__main__':
+  price: int
+  for price in prices:
+    if price > high:
+      high = price
+    elif price < low:
+      low = price
+
+  return high - low
```

Comment:
Unfortunately this algorithm isn't so simple, This function should be buying and selling prices for the highest profit, the catch is it "can see the future". Its not as simple as buying at the lowest price and selling at the highest price because you can only sell at a price that comes after the price that you buy (cant go back in time).